extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -162,6 +162,8 @@ export abstract class DacFxConfigPage extends BasePage {
 		this.fileTextBox.ariaLabel = localize('dacfx.fileLocationAriaLabel', "File Location");
 		this.fileButton = this.view.modelBuilder.button().withProperties({
 			label: '•••',
```

Comment:
should we use … (ellipsis) or 3 dots?

Reply:
this is what Smitha told us use when we first made this wizard.